Jeremy_Kracy/Database/main.py

```diff
@@ -0,0 +1,36 @@
+import json
+
+
+def read_in(file_name):
+    file = open(file_name, 'r')
+    data = json.load(file)
+
+    print('CREATE TABLE Customer_Information (\n'
+          'Email varchar(50) NOT NULL,\n'
+          'Name varchar(50),\n'
+          'Cell_Phone varchar(10),\n'
+          'Work_Phone varchar(10),\n'
+          'Address varchar(50),\n'
+          'PRIMARY KEY (Email)\n'
```

Comment:
What was your thought process in deciding what primary key to use for Customer_Information table? And what kind of relationship do you anticipate between the two tables (Customer_Information and Order_Information)?

Reply:
I decided to use the customer's email as the primary key in the Customer_Information table, because it seemed to be the best characteristic to use as an identifier. I think of other account based services that will reject accounts if the email used is already taken by a different account. That was the idea I was going for with that. I anticipate a one-to-many relationship for Customer_Information and Order_Information. This would allow 1 customer to have multiple orders, while ensuring each order only corresponds to one customer.